src/ui/components/Auth/Login.jsx

```diff
@@ -0,0 +1,55 @@
+import './auth.css';
+import React, { memo, useCallback, useRef } from 'react';
+import { TextField, Button } from '@material-ui/core';
+import axios from 'axios';
+import { useQueryCache } from 'react-query';
+
+const Login = () => {
+  const cache = useQueryCache();
+  const emailTextField = useRef();
+  const passwordTextField = useRef();
+
+  const onSubmit = useCallback(() => {
+    if (emailTextField.current?.value && passwordTextField.current?.value) {
+      axios
+        .post(
+          `/auth/login`,
+          {
+            email: emailTextField.current.value,
+            password: passwordTextField.current.value
+          },
+          { withCredentials: true }
+        )
+        .then(() => {
+          cache.invalidateQueries('authorized-user');
+        })
+        .catch(err => console.log(err));
```

Comment:
Not sure how much we care about the UI, but this could display on the UI if the authorization fails.  Feels weird clicking "login" and having nothing happen.

Reply:
Added a snackbar alert. I kept the console.error. Let me know if you think this should be deleted